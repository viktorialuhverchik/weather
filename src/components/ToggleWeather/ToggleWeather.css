.toggle-weather_container {
    margin: 2vh 0;
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;

}

.app-link, .app-link:hover {
    text-decoration: none;
    color: #1a2ab4;
}

.today{
    margin-right: 10vw;
}

.active {
    text-decoration: underline;
}