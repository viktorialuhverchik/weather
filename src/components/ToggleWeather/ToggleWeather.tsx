import React, { FC } from 'react';
import { Link } from 'react-router-dom';

import './ToggleWeather.css';

const ToggleWeather: FC = () => {
    
    return (
        <div className="toggle-weather_container ">
            <Link to="/" className={`app-link today`}>Today</Link>
            <Link to="/fivedays" className={`app-link`}>Five days</Link>
        </div>
    );
};
export default ToggleWeather;
