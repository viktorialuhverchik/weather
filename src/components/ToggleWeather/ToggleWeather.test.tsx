import React from 'react';
import ToggleWeather from './ToggleWeather';
import { renderWithRedux } from '../../index.test';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect'


describe('ToggleWeather component',() => {
    it('should render ToggleWeather component', () => {
        const wrapper = renderWithRedux(<ToggleWeather/>);
        expect(wrapper).toMatchSnapshot();
    });

    it('includes 2 link', () => {                                       
        const wrapper = shallow(<ToggleWeather/>);
        expect(wrapper.find('Link').length).toBe(2);
    });

});