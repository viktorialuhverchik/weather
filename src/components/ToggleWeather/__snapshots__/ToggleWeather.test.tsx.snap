// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleWeather component should render ToggleWeather component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <script
      defer=""
      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCVTVRvhts70T-KlhGw14mejDBAVFTlb7w&libraries=places"
      type="text/javascript"
    />
    <div>
      <div
        class="toggle-weather_container "
      >
        <a
          class="app-link today"
          href="/"
        >
          Today
        </a>
        <a
          class="app-link"
          href="/fivedays"
        >
          Five days
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="toggle-weather_container "
    >
      <a
        class="app-link today"
        href="/"
      >
        Today
      </a>
      <a
        class="app-link"
        href="/fivedays"
      >
        Five days
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
